core/cmd/client.go

```diff
@@ -62,7 +62,7 @@ var (
 type Client struct {
 	Renderer
 	Config                         config.GeneralConfig
-	Logger                         logger.Logger
+	Logger                         logger.CloseableLogger
```

Comment:
Maybe just bike shedding style at this point, but I wonder if `logger.CloseableLogger` is still helpful, or if it would be simpler to do:
```suggestion
	Logger                         logger.Logger
	LoggerClose                    func() error // May be nil.
```
I usually prefer nil to no-op in the spirit of "Make the zero value useful." https://go-proverbs.github.io/